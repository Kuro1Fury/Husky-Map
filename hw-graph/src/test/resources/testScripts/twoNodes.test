# Two nodes test
# Tests that a graph with two nodes can be generated

CreateGraph A
AddNode A n1
AddNode A n2