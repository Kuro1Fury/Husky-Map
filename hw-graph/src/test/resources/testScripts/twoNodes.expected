# Two nodes test
# Tests that a graph with two nodes can be generated

created graph A
added node n1 to A
added node n2 to A
